shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.bat

```diff
@@ -27,6 +27,13 @@ set PORT=%1
 
 set CONFIG=%2
 
+if "%PORT%"=="-h" (
```

Comment:
Why .bat is different from .sh and use goto?

Reply:
windows bat is a little difference with bash,  it use goto as a function call